indigo_app/templates/indigo_api/document/show.html

```diff
@@ -37,18 +36,28 @@ <h5 class="main-header-title">
               {% if request.user.is_authenticated %}
                 {% if user_can_edit %}
                   <div class="btn-group save-btn-group">
-                    <button class="btn btn-success save" disabled><span class="if-published">Save &amp; publish</span><span class="if-draft">Save draft</span></button>
+                    <button class="btn btn-success save" disabled>
+                      <span class="if-published">{% trans 'Save &amp; publish' %}</span>
```

Comment:
will `&amp;` be fine here?

Reply:
It seems this is the correct way to do it :/